x/leverage/keeper/keeper.go

```diff
@@ -531,7 +531,7 @@ func (k Keeper) LiquidateBorrow(
 		}
 	}
 
-	// Detect bad debt (collateral == 0 after reward) for repayment by reserves next InterestEpoch
+	// Detect bad debt (collateral == 0 after reward) for repayment by protocol reserves (see ADR-004)
```

Comment:
```suggestion
	// Detect bad debt (collateral == 0 after reward) for repayment by protocol reserves
```

Reply:
Why you want to remove note for ADR-004? I think it's good to reference it.

Reply:
004 Doesn't really deal with bad debt. 007 reference would be more appropriate if we need one

Reply:
It's about protocol reserves. They are defined in ADR-004